src/Lexer.js

```diff
@@ -388,6 +389,14 @@ module.exports = class Lexer {
         continue;
       }
 
+      // get previous text char
+      if (tokens.length > 0 && tokens[tokens.length - 1].type === 'text') {
+        lastTokenRaw = tokens[tokens.length - 1].raw;
+        prevChar = lastTokenRaw.charAt(lastTokenRaw.length - 1);
+      } else {
+        prevChar = '';
+      }
```

Comment:
I wonder if there's a way to do this without introducing this extra processing of the tokens every loop?

What about something dumb like this with a flag `keepPrevChar`:

```
inlineTokens(src, tokens = [], inLink = false, inRawBlock = false) {
...
  let keepPrevChar, prevChar;
...

while (src) {
  if(!keepPrevChar) {
    prevChar = '';
  }
  keepPrevChar = false;

....

  if (token = this.tokenizer.inlineText(src, inRawBlock, smartypants)) {
    src = src.substring(token.raw.length);
    prevChar = token.raw.slice(-1);
    keepPrevChar = true;
    tokens.push(token);
    continue;
  }
...
}
}
```

Reply:
@calculuschild Seems like a good solution 👍

Reply:
I updated it :+1: